src-java/floodlight-service/floodlight-modules/src/main/java/org/openkilda/floodlight/command/flow/ingress/IngressFlowSegmentBase.java

```diff
@@ -209,35 +259,35 @@ private void handleMeterRemoveReport(MeterRemoveReport report) {
         }
     }
 
-    protected List<OFFlowMod> makeFlowModMessages(MeterId effectiveMeterId) {
+    protected List<OFFlowMod> makeFlowModMessages(MeterId effectiveMeterId, GroupId effectiveGroupId) {
```

Comment:
Why not just use `EffectiveIds` as argument?

Reply:
Now we use `EffectiveIds` as argument